src/coreclr/gc/gc.cpp

```diff
@@ -30281,6 +30281,15 @@ void gc_heap::mark_phase (int condemned_gen_number)
         drain_mark_queue();
         fire_mark_event (ETW::GC_ROOT_HANDLES, current_promoted_bytes, last_promoted_bytes);
 
+#ifdef FEATURE_GCBRIDGE
+        dprintf(3,("GCBridge to mark handles"));
+        GCScan::GcScanWithBridge(GCHeap::Promote,
+                                    condemned_gen_number, max_generation,
+                                    &sc);
+        drain_mark_queue();
```

Comment:
wonder whether `drain_mark_queue` is required after each scan or can it be combined with the one above ?